src/test/java/com/synopsys/integration/blackduck/bdio2/Bdio2RetryAwareStreamUploaderTest.java

```diff
@@ -15,17 +15,44 @@
 class Bdio2RetryAwareStreamUploaderTest {
 
     @Test
-    void testStartRetriable() throws IntegrationException {
+    void testStartRetriable() throws IntegrationException, InterruptedException {
         BlackDuckRequestBuilderEditor editor = Mockito.mock(BlackDuckRequestBuilderEditor.class);
         BdioFileContent bdioFileContent = Mockito.mock(BdioFileContent.class);
         Bdio2RetryAwareStreamUploader bdio2RetryAwareStreamUploader = mockBdio2RetryAwareStreamUploaderThrows512OnStart(editor, bdioFileContent);
         try {
-            bdio2RetryAwareStreamUploader.start(bdioFileContent, editor);
+            bdio2RetryAwareStreamUploader.start(bdioFileContent, editor, System.currentTimeMillis(), System.currentTimeMillis() + 10000);
             Assertions.fail("Expected RetriableBdioUploadException");
         } catch (RetriableBdioUploadException e) {
             // expected
         }
     }
+    
+    @Test
+    void testStartRetrableWithRetryAfterHeader() throws IntegrationException, RetriableBdioUploadException, InterruptedException {
```

Comment:
Not a big deal just a spelling error in the method name I believe
```suggestion
    void testStartRetryableWithRetryAfterHeader() throws IntegrationException, RetriableBdioUploadException, InterruptedException {
```